File: Ad-Hoc/URI 1893.ml
let inicio, final = Scanf.scanf "%i %i" (fun a b -> a, b);;

if (inicio >= 0 && final <= 2) then (
    Printf.printf "nova\n";
) else if (final > inicio && final <= 96) then (
    Printf.printf "crescente\n";
) else if (inicio >= final && final <= 96) then (
    Printf.printf "minguante\n";
) else ( 
    Printf.printf "cheia\n";
)

File: Ad-Hoc/URI 1893.go
package main
 
import (
    "fmt"
)
 
func main() {
    var inicio, final int

    fmt.Scanf("%d", &inicio)
    fmt.Scanf("%d", &final)

    if (inicio >= 0 && final <= 2) {
        fmt.Println("nova")
    } else if (final > inicio && final <= 96) {
        fmt.Println("crescente")
    } else if (inicio >= final && final <= 96) {
        fmt.Println("minguante")
    } else { 
        fmt.Println("cheia")
    }
}